daras_ai_v2/sarvam_asr.py

```
@@ -0,0 +1,150 @@
+    *,
+    audio_url: str,
+    language_code: str,
+    mode: str,
+    with_timestamps: bool,
+) -> dict:
+    if not settings.SARVAM_API_KEY:
+        raise UserError(
+            "Sarvam AI ASR is not configured: missing SARVAM_API_KEY"
+        )
+
+    audio_response = requests.get(audio_url)
```

Ruff (0.16.0)

[error] Probable use of `requests` call without timeout

(S113)